report_qweb_signer/models/ir_actions_report.py

```diff
@@ -17,16 +17,6 @@
 _logger = logging.getLogger(__name__)
 
 
-def _normalize_filepath(path):
-    path = path or ""
-    path = path.strip()
-    if not os.path.isabs(path):
```

Comment:
but it did already support absolute paths before ? So why removing the possibility to use relative path ?